BonusCalcListener.Tests/UseCase/PayElementMapperTests.cs

```diff
@@ -244,6 +319,28 @@ public void ShouldCreateOvertimePayElementWithZeroValue()
             result.Value.Should().Be(0.0m);
         }
 
+        [Test]
+        public void WhenPaymentTypeIsNotOvertime_ShouldCreateReactivePayElement()
+        {
+            // Arrange
+            var evtData = _fixture.Create<WorkOrderOperativeSmvData>();
+            evtData.WorkOrderStatusCode = 50;
+            evtData.StandardMinuteValue = 0;
+            evtData.JobPercentage = 50;
+            evtData.ClosedTime = new DateTime(2021, 11, 08, 14, 32, 01);
+            evtData.IsOutOfHours = false;
+            //evtData.IsOvertime = true;
```

Comment:
Should this be explicitly set to false as that's the column default in Repairs Hub?

Reply:
Yes, but the isOvertime value is ignored when paymentType is not null. I have added testcases that set isOvertime to true and false to show it has no effect. Plus isOvertime will eventually be removed after migration.